DataFormats/CTPPSDigi/interface/CTPPSPixelDigiCollection.h

```diff
@@ -0,0 +1,31 @@
+#ifndef CTPPS_TRACKERDIGICOLLECTION_H
```

Comment:
Please use an include guard matching the full file name subsystem_package_fileName_h